<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeFinderConfig</class>
 <widget class="QWidget" name="RangeFinderConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>136</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h2&gt;Range Finder&lt;/h2&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../qgroundcontrol.qrc">:/files/images/devices/AC-0004-11-2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Reading</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Range (m):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rangeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Volatge:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="voltageLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>346</width>
     <height>335</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="typeComboBox"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="scalingEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Scaling m/V:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Min Distance (cm):</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Stop Pin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Function:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="functionComboBox"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="maxDistanceEdit"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSlider" name="gainSlider">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Settle time (ms):</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="settleTimeEdit"/>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="inputPinEdit"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Input Pin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="minDistanceEdit"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="offsetEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Max Distance (cm):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Gain:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLineEdit" name="stopPinEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Offset (V):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="gainLabel">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="rmetricCheckBox">
      <property name="text">
       <string>Ratio Metric</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>functionComboBox</tabstop>
  <tabstop>rmetricCheckBox</tabstop>
  <tabstop>gainSlider</tabstop>
  <tabstop>maxDistanceEdit</tabstop>
  <tabstop>minDistanceEdit</tabstop>
  <tabstop>offsetEdit</tabstop>
  <tabstop>scalingEdit</tabstop>
  <tabstop>settleTimeEdit</tabstop>
  <tabstop>inputPinEdit</tabstop>
  <tabstop>stopPinEdit</tabstop>
  <tabstop>rangeLineEdit</tabstop>
  <tabstop>voltageLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
